README.md

```diff
@@ -91,16 +91,41 @@ Results will be added to different tagged releases along with the specific confi
 
 ## Testing
 
-Unit tests can be written for the _reference system_ to check if all nodes, topics and other requirements are met before accepting test results.
+Tests can be written for the _reference system_ to check the platform as well as if all nodes, topics and other requirements are met before accepting test results.
 
 Tests are provided to automatically generate results for you by running `colcon test` on a supported platform above.
 
-To run the test, simply run the following command from your workspace:
+**Note:** during the testing _trace data_ generated from `LTTng` will be placed in `colcon_ws/tracing`. If the tracing directory isnt there the tests will automatically generate it for you.
```

Comment:
is it really under `colcon_ws/tracing`? Traces should go under `~/.ros/tracing/` by default.